src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -54,6 +54,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- If targeting the same release that is bundled with the .NET Core SDK, use the bundled package version provided by Microsoft.NETCoreSdk.BundledVersions.props -->
     <NETStandardImplicitPackageVersion Condition="'$(NETStandardImplicitPackageVersion)' =='' And '$(_TargetFrameworkVersionWithoutV)' == '$(BundledNETStandardTargetFrameworkVersion)'">$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
 
+	<!-- If targeting .NET Standard 2.0.x, use version 2.0.0 of the package. -->
+    <NETStandardImplicitPackageVersion Condition="'$(NETStandardImplicitPackageVersion)' =='' And '$(_TargetFrameworkVersionWithoutV)' &gt;= '2.0' And '$(_TargetFrameworkVersionWithoutV)' &lt; '2.1'">2.0.0</NETStandardImplicitPackageVersion>
+
```

Comment:
I think this should go before the other clause and just match on == 2.0. There should be symmetry between this and the .NET Core app case.

Reply:
Actually, hang on.

Reply:
Looks like the intent was to change the final clause to 2.0.0 now. See the comment on line 62 before/65 after. "Currently this is the same as the previous clause, but when we have a stable 2.0 package this should change."

Reply:
@dsplaisted

Reply:
Yes, I think @nguerrera is right, and the logic we want is:

- If targeting bundled target framework version of .NET Standard, then use the bundled version of the package
- If targeting a version of .NET Standard less than 2.0, then use version 1.6.1 of the package.
- Otherwise, use the latest stable version of the package (currently 2.0.0)

This means that when there's eventually a .NET Standard 2.1 and a corresponding stable package, we would reference package version 2.1.0 even when targeting .NET Standard 2.0.  The reason we don't do that for 1.x is because the reference ends up getting exposed as a NuGet dependency when you pack the project, but we stopped doing that for 2.0.

@ericstj @weshaggard does the logic I've outlined sound right?

Reply:
I see. It makes sense. I will change the default number to 2.0.0.

Reply:
Any reason not to use `2.0.1`? https://github.com/dotnet/standard/pull/492.
With out it, it will never be used on mono (no bundled version), but it only contains an UWP fix and an xbuild workaround. But https://github.com/dotnet/standard/pull/510 will be important for perf improvements.

Reply:
@dasMulli Good point, it should probably be 2.0.1 for now, and updated again once there is a package with https://github.com/dotnet/standard/pull/510